protocol/src/test/kotlin/tbdex/sdk/protocol/TestData.kt

```diff
@@ -32,8 +33,7 @@ import web5.sdk.dids.Did
 import web5.sdk.dids.DidKey
 import java.net.URI
 import java.time.OffsetDateTime
-import java.util.Date
-import java.util.UUID
+import java.util.*
```

Comment:
Do we have a lint setting to prefer * imports?

Reply:
I can't figure out how to make it stop this!